lib/razor/cli/command.rb

```diff
@@ -41,8 +41,10 @@ def extract_command
     body = {}
     until @segments.empty?
       argument = @segments.shift
-      if argument =~ /\A--([a-z-]+)(=(.+))?\Z/
-        # `--arg=value` or `--arg value`
+      if argument =~ /\A--([a-z-]{2,})(=(.+))?\Z/ or
+          argument =~ /\A-([a-z])(=(.+))?\Z/
+        # `--arg=value`/`--arg value`
+        # `-a=value`/`-a value`
```

Comment:
I don't know how many short flags that don't take arguments razor supports; if there's not very many then the utility of combining multiple short flags might not be worth the decreased error ergononimcs.